### ISSUE TITLE
can hmmlearn be used in an exe?

### ISSUE THREAD

**Issue by ConfitureCheung** [hmmlearn/hmmlearn#396]
I am trying to convert .py to .exe and hmmlearn is one of the packages in my .py
i have tried many times and still don't work. Throughout this trial and error process it seems to me that hmmlearn is the issue that cause the exe version not working. 

i would just like to confirm if this is true. thanks in advance.

**Comment by anntzer** [comment]
This is likely a problem on your exe generator (which you have not specified), not on hmmlearn's side.  Please report to them, and link back their reply if they disagree.

**Comment by ConfitureCheung** [comment]
I have tried both CX_Freeze, Pyinstaller and py2exe converter. With seeing your reply, I tried using the example code from hmmlearn doc: 
https://hmmlearn.readthedocs.io/en/latest/tutorial.html. 
>>> import numpy as np
>>> from hmmlearn import hmm
>>> np.random.seed(42)
>>>
>>> model = hmm.GaussianHMM(n_components=3, covariance_type="full")
>>> model.startprob_ = np.array([0.6, 0.3, 0.1])
>>> model.transmat_ = np.array([[0.7, 0.2, 0.1],
...                             [0.3, 0.5, 0.2],
...                             [0.3, 0.3, 0.4]])
>>> model.means_ = np.array([[0.0, 0.0], [3.0, -3.0], [5.0, 10.0]])
>>> model.covars_ = np.tile(np.identity(2), (3, 1, 1))
>>> X, Z = model.sample(100)

again, it works in my IDE (Pycharm) but not working after converting to exe. in CX_Freeze version, it mentioned line 2 of my test.py (i.e. from hmmlearn import hmm)

in Pyinstaller version, it mentioned Fatal Python error: Py_Initialize: unable to load the file system codec
LookupError: unknown encoding: utf-8

Current thread 0x00003ef8 (most recent call first):
